src/rsa.rs

```diff
@@ -60,61 +60,67 @@ struct RSAVerificationAlgorithm {
 }
 
 
-impl signature_impl::VerificationAlgorithmImpl for RSAVerificationAlgorithm {
-    fn verify(&self, public_key: untrusted::Input, msg: untrusted::Input,
-              signature: untrusted::Input) -> Result<(), ()> {
-        const MAX_BITS: usize = 8192;
-
-        let (n, e) = try!(parse_public_key(public_key));
-        let signature = signature.as_slice_less_safe();
+fn verify(padding: &RSAPadding, msg: untrusted::Input,
```

Comment:
I would rather just call this `verify_rsa` so it is easier to find. Most people will be looking for it under that name. Also, we need to add the end-user documentation for this function.